libbeat/_meta/fields.ecs.yml

```diff
@@ -2261,10 +2261,6 @@
       type: keyword
       description: >
         Major version of the user agent.
-    - name: user_agent.device
-      type: keyword
-      description: >
-        Name of the physical device.
```

Comment:
This field def overrides the "real" ECS field, defined on line [2180](https://github.com/elastic/beats/pull/10472/files#diff-c2a118728ec0aba85808c297833ade48L2180)

Reply:
:-(